tokio-util/src/context.rs

```diff
@@ -0,0 +1,38 @@
+//! This module introduces `TokioContext`, which enables a simple way of using
```

Comment:
May want to also consider adding a `HandleExt` trait extension that could provide a `wrap` fn that produces a `TokioContext` from a handle.

```rust
trait HandleExt {
	fn wrap<F: Future>(f: F) -> TokioContext<F> {...}
}

impl HandleExt for Handle {}
```